docs/cli/data.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: data command
+parent: SMT CLI
+nav_order: 2
+---
+
+# Data subcommand
+{: .no_toc }
+
+This subcommand will perform data migration in either the POC or the minimal downtime mode. This subcommand requires users to pass the session file (which contains schema mapping) generated by either the `schema` subcommand or web UI.
+
+{: .highlight }
+The command below assumes that the open-source version of SMT is being used. For the CLI
+reference of the gCloud version of SMT, please refer [here](https://cloud.google.com/sdk/gcloud/reference/alpha/spanner/migration).
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+## NAME
+
+    ./spanner-migration-tool - migrate data from a source database to
+        Cloud Spanner given a schema
+
+## SYNOPSIS
+
+    ./spanner-migration-tool --session=SESSION --source=SOURCE
```

Comment:
The commands on this page missin data subcommand?

Reply:
Thanks, fixed.